.github/workflows/release-libs.yaml

```diff
@@ -32,113 +28,91 @@ jobs:
             override: true
       - name: Login
         run: cargo login ${{ secrets.CRATES_IO_DEPLOY_KEY }}
+
       - name: Publish crate common
-        continue-on-error: true
         run: |
-          cd common
-          cargo publish
+          ./scripts/release-libs.sh common
```

Comment:
I suggest to move all the `./scripts/release-libs.sh [crate-name]` calls to a shell script with a for loop instead of the repeated code here.